data/talkactions/scripts/create_item.lua

```diff
@@ -8,10 +8,17 @@ function onSay(player, words, param)
 	end
 
 	local split = param:split(",")
+	if not split[1] then
```

Comment:
Yeah I considered that but I thought it felt somewhat hackish doing that. But I guess you're right.